assets/dev/js/editor/views/add-section/base.js

```diff
@@ -9,7 +9,9 @@ import environment from 'elementor-common/utils/environment';
 
 	// Views.
 	static VIEW_CHOOSE_ACTION = 'choose-action';
-	static VIEW_SELECT_PRESET = ( AddSectionBase.IS_CONTAINER_ACTIVE ) ? 'select-container-preset' : 'select-preset';
+	static VIEW_CONTAINER_FLEX = 'select-container-preset';
```

Comment:
I am thinking could you use the 'preset' here as well. Is that helpful?
Like `VIEW_CONTAINER_FLEX_PRESET`.